modules/update-upgrading-oc-adm-upgrade-recommend.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * updating/updating_a_cluster/updating-cluster-cli.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="update-upgrading-oc-adm-upgrade-recommend_{context}"]
+= Finding recommended update paths with `oc adm upgrade recommend` (Technology Preview)
```

Comment:
It seem this style is a little different from `Gathering cluster update status using oc adm upgrade status (Technology Preview)`, could you please help take a look?

Reply:
Hi @JianLi-RH ! Happy to take a look. What specifically are you referring to?

Reply:
hi @obrown1205, From the preview page, I can see `oc adm upgrade recommend` is shown as a code block, but `oc adm upgrade status` is not. And because I can't see the backend code, so need you confirm. Thanks!

Reply:
Ah! Fixed it :) Can you PTAL?